modules/installation-configuration-parameters.adoc

```diff
@@ -1108,15 +1103,48 @@ link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
+|Any license available with the link:https://cloud.google.com/compute/docs/reference/rest/v1/licenses/list[license API], such as the license to enable link:https://cloud.google.com/compute/docs/instances/nested-virtualization/overview[nested virtualization]. You cannot use this parameter with a mechanism that generates pre-built images. Using a license URL forces the installation program to copy the source image before use.
 
-|`platform.gcp.osDisk.diskSizeGB`
+|`platform.gcp.defaultMachinePlatform.zones`
+|The availability zones where the installation program creates machines.
+|A list of valid link:https://cloud.google.com/compute/docs/regions-zones#available[GCP availability zones], such as `us-central1-a`, in a
+link:https://yaml.org/spec/1.2/spec.html#sequence//[YAML sequence].
+
+|`platform.gcp.defaultMachinePlatform.osDisk.diskSizeGB`
 |The size of the disk in gigabytes (GB).
 |Any size between 16 GB and 65536 GB.
 
-|`platform.gcp.osDisk.diskType`
-|The type of disk.
-|Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. The worker nodes can be either type.
+|`platform.gcp.defaultMachinePlatform.osDisk.diskType`
+|The link:https://cloud.google.com/compute/docs/disks#disk-types[GCP disk type].
+|Either the default `pd-ssd` or the `pd-standard` disk type. The control plane nodes must be the `pd-ssd` disk type. Compute nodes can be either type.
+
+|`platform.gcp.defaultMachinePlatform.tags`
+|Optional. Additional network tags to add to the control plane and compute machines.
+|One or more strings, for example "network-tag1".
+
+|`platform.gcp.defaultMachinePlatform.type`
+|The link:https://cloud.google.com/compute/docs/machine-types[GCP machine type] for control plane and compute machines.
+|The GCP machine type, for example "n1-standard-4". 
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.name`
+|The name of the customer managed encryption key to be used for machine disk encryption.
+|The encryption key name.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.keyRing`
+|The name of the Key Management Service (KMS) key ring to which the KMS key belongs.
+|The KMS key ring name.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.location`
+|The link:https://cloud.google.com/kms/docs/locations[GCP location] in which the KMS key ring exists.
+|The GCP location.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.projectID`
+|The ID of the project in which the KMS key ring exists. This value defaults to the value of `platform.gcp.projectID` if not set.
+|The GCP project ID.
+
+|`platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKey.kmsKeyServiceAccount`
```

Comment:
It should be `platform.gcp.defaultMachinePlatform.osDisk.encryptionKey.kmsKeyServiceAccount` instead.

Reply:
Good catch, thanks.